src/main/resources/db/migration/mysql/V1.6.1__add-failed-attempt-counter.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `identity` ADD `failed_login_attempts` INT NOT NULL DEFAULT 0;
```

Comment:
can we rename the file to `V1.6.1__add-failed-login-attempt-counter`

Reply:
sure